### DISCUSSION TITLE
✅ Add rem unit for padding in icon and icon box widgets

### DISCUSSION THREAD

**Discussion by bytepack** [elementor/elementor#23322] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

If we use rem unit in some elements and px in others, it could break the UI when browser font size changes.
I want to be able to use rem unit in Elementor icon and icon box widgets in **_style > icon > padding_** without custom CSS.

### Describe the solution you'd like

In icon and icon box widgets, in **_style > icon > padding_**, only px unit is allowed.
Please add rem and other units.
Thanks.

**Comment by m1ga** [comment]
where do you see `style > icon > padding` ? In the Icon Box widget I only have `Spacing` and it allows you to switch to REM or even add custom styles.

**Reply by bytepack** [reply]
In **_Content_** tab, when **_View_** is changed to **_Stacked_** or **_Framed_**,  **_Padding_** will appear in **_Style_** tab.

**Comment by rami-elementor** [comment]
Already exists:

![image](https://github.com/user-attachments/assets/76199f86-b6a8-4786-a129-d450931e0ef9)

![image](https://github.com/user-attachments/assets/c5ceceb2-dbe9-4b12-a5ac-5e8b56f84694)

**Comment by nicholaszein** [comment]
Hello there!

As mentioned by @rami-elementor, 

### 📢 We're happy to announce that this feature was released with Elementor v3.19.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
